js/index.js

```diff
@@ -39,4 +39,137 @@ const siteContent = {
+navItems[1].textContent = siteContent["nav"]["nav-item-2"];
+navItems[2].textContent = siteContent["nav"]["nav-item-3"];
+navItems[3].textContent = siteContent["nav"]["nav-item-4"];
+navItems[4].textContent = siteContent["nav"]["nav-item-5"];
+navItems[5].textContent = siteContent["nav"]["nav-item-6"];
+
+navItems.forEach(element => {
+  element.style.color = "green";
+  element.style.fontFamily = "Schoolbell";
+});
+
+let newItemOne = document.createElement("a");
+newItemOne.textContent = "Dog";
+newItemOne.style.fontFamily = "Schoolbell";
+
+let newItemTwo = document.createElement("a");
+newItemTwo.textContent = "muffin";
+newItemTwo.style.fontFamily = "Schoolbell";
+
+let navNavNav = document.querySelector("nav");
+navNavNav.append(newItemTwo);
+newItemTwo.style.color = "green";
+navNavNav.prepend(newItemOne);
+newItemOne.style.color = "green";
+navNavNav.style.justifyContent = "space-between";
+
+
+//cta
+let titleText = document.querySelector("h1");
+let buttonGadget = document.querySelector("button");
+let heroImage = document.getElementById("cta-img");
+let colorBackground = document.querySelector("body");
+
+titleText.textContent = siteContent["cta"]["h1"];
+titleText.style.fontFamily = "Barriecito";
+buttonGadget.textContent = siteContent["cta"]["button"];
+heroImage.src = siteContent["cta"]["img-src"];
+
+titleText.addEventListener("mouseenter", () => {
+  titleText.textContent = "Manipulate that DOM!!!";
+});
+
+titleText.addEventListener("mouseleave", () => {
+  titleText.textContent = siteContent["cta"]["h1"];
+})
+
+const shenanigan = document.createElement("h1");
+shenanigan.textContent = "Why not make it ridiculous?";
+const cta = document.querySelector(".cta");
+const ctaText = document.querySelector(".cta-text");
+
+buttonGadget.addEventListener("click", () => {
+  cta.append(shenanigan);
+  titleText.style.color = "red";
+  colorBackground.style.backgroundColor = "lime";
+  ctaText.style.width = "45%";
+  shenanigan.style.fontFamily = "Barriecito";
+  shenanigan.style.padding = "50px";
+  shenanigan.style.fontSize = "60px";
+  shenanigan.style.backgroundColor = "pink";
+  shenanigan.style.width= "100%";
+  shenanigan.style.textAlign = "center";
+  cta.style.flexWrap = "wrap";
+  
+  navItems.forEach(element => {
+    element.style.fontFamily = "Barriecito";
+  }); //end forEach
+
+});//end button gadget
+
+const newButton = document.createElement("button");
+newButton.textContent = "back to professional please!";
+const daFooter = document.querySelector("footer");
+daFooter.append(newButton);
+newButton.addEventListener("click", () => {
+  titleText.style.color = "black";
+  colorBackground.style.backgroundColor = "white";
+  shenanigan.style.display = "none";
+  navItems.forEach(element => {
+    element.style.fontFamily = "Schoolbell";
+  });//end forEach
+}); //end button
+
+heroImage.addEventListener("mouseenter", () => {
+  heroImage.src = "../img/whoodle-pup.jpg";
+});
+
```

Comment:
Since this is going to be inserted into `index.html` which is at the root of the project. You need to leave off the two dots at the beginning of the path in order to access the `img` folder which is at the same level as `index.html`.